# Request
bool request
uint8 release
---
# Response
uint8 worker_number